integration-tests/oidc-code-flow/src/main/resources/application.properties

```diff
@@ -204,3 +204,9 @@ quarkus.log.category."io.quarkus.vertx.http.runtime.security.HttpAuthenticator".
 quarkus.log.category."io.quarkus.vertx.http.runtime.security.HttpSecurityRecorder".level=DEBUG
 
 quarkus.log.category."com.gargoylesoftware.htmlunit.javascript.host.css.CSSStyleSheet".level=FATAL
+
+# make code flow default for all paths expect for 'test-security-annotation' path to test annotations
+quarkus.http.auth.permission.use-code-flow-by-default.paths=/web-app*,/web-app2*,/web-app3*,/tenant-autorefresh*,/tenant-https*,/tenant-logout*,/tenant-nonce*,/tenant-refresh*,/public-web-app*,/index.html,/,/tenant-cookie-path-header,/tenant-javascript
```

Comment:
I'm a little bit concerned about it, as it introduces a new direction into how the whole set of tests is running, and we have this big list of paths. I'd rather add a dedicated `extensions/oidc/deployment` test with a clear definition of what is tested.